src/resolvers/User.js

```diff
@@ -75,7 +88,17 @@ async function events(parent, { association, status }, context) {
 }
 
 async function ownedEvents(parent, args, context) {
-  return events(parent, { association: ['OWNER'], status: ['SET', 'LIT', 'COMPLETED'] }, context);
+  const requestingUserId = await getUserId({ context });
+
+  // Can only view your own owned events
+  if (requestingUserId !== parent.id) {
+    throw new AuthenticationError('Not authorized to view owned events');
+  }
+
+  return events(parent, {
+    association: [ASSOCIATION.owner],
+    status: [EVENT_STATUS.set, EVENT_STATUS.lit, EVENT_STATUS.completed],
+  }, context);
 }
 
 export default {
```

Comment:
All of the fields without an explicit resolver in User.js will be publicly accessible to anyone with a user id. That's ok as long as that's an active decision being made

Reply:
yes. Everything else is meant to be public. Those fields are

id, name, email, bio, score.

An argument can be made about email tho